src/GitHub.TeamFoundation.14/Home/GitHubHomeSection.cs

```diff
@@ -93,12 +76,24 @@ bool IsGitToolsMessageVisible()
                 RepoName = ActiveRepoName;
                 RepoUrl = ActiveRepoUri.ToString();
                 Icon = GetIcon(false, true, false);
+               
+                var weekElapsed = settings.WelcomeMessageLastSeen.GetIso8601WeekOfYear() != DateTimeOffset.Now.GetIso8601WeekOfYear();
```

Comment:
Since `GetIso8601WeekOfYear()` returns an int I could just change that to compare
`Current.GetIso8601WeekOfYear() > WelcomeMessageLastSeen.GetIso8601WeekOfYear() = showmessage`?